src/main/scala/cromwell/engine/backend/jes/OperationMetadata.scala

```diff
@@ -14,11 +13,18 @@ object OperationMetadata {
       metadata.get("endTime").map(_.asInstanceOf[String].toDate))
   }
 
-  val Iso8601DateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+  val defaultDate = DateTime.parse("0")
```

Comment:
True... perhaps the egregious lie is actually preferable as it will jump out as a lie, while "now" would not.